speculation-rules.bs

```diff
@@ -368,7 +377,12 @@ add the following step
   1. If |input|["`target_hint`"] [=map/exists=]:
     1. If |input|["`target_hint`"] is not a [=valid navigable target name or keyword=], then return null.
     1. Set |targetHint| to |input|["`target_hint`"].
-  1. Return a [=speculation rule=] with [=speculation rule/URLs=] |urls|, [=speculation rule/predicate=] |predicate|, [=speculation rule/requirements=] |requirements|, and [=speculation rule/target navigable name hint=] |targetHint|.
+  1. Let |referrerPolicy| be the empty string.
+  1. If |input|["`referrer_policy`"] [=map/exists=]:
+    1. If |input|["`referrer_policy`"] is not a [=referrer policy=], then return null.
```

Comment:
What if it's not a sufficiently-strict referrer policy? Probably that should return null too?

Reply:
Since we'll allow other policies for the same site case, I don't think we should enforce that here. Even for the cross-site case, it doesn't really matter if we accept it here, since the rule will still be ignored when we check the computed referrer policy.

Reply:
Great point.